content/sensu-go/6.3/operations/manage-business-services/components.md

```diff
@@ -0,0 +1,522 @@
+required     | true
+type         | Map of key-value pairs
+example      | {{< language-toggle >}}
+{{< code yml >}}
+rules:
+  - template: status-threshold
+    arguments:
+      status: non-zero
+      threshold: 25
+{{< /code >}}
+{{< code json >}}
+{
+  "rules": [
+    {
+      "template": "status-threshold",
+      "arguments": {
+        "status": "non-zero",
+        "threshold": 25
+      }
+    }
+  ]
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+handlers     | 
+-------------|------ 
+description  | List of handlers to use for the events the component produces. The component will set the handlers property in events that are produced by rule evaluation. If no handlers are specified in the component definition, handlers can be set by the monitoring rule itself via template arguments. Handlers specified in the component definition will override any handlers set by rule evaluation.
+required     | false
+type         | Array
+example      | {{< language-toggle >}}
+{{< code yml >}}
+handlers:
+  - pagerduty
+  - slack
+{{< /code >}}
+{{< code json >}}
+{
+  "handlers": [
+    "pagerduty",
+    "slack"
+  ]
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+#### Query attributes
+
+type         | 
+-------------|------ 
+description  | Type of selector to use to identify the events that the monitoring rule should process: `fieldSelector` or `labelSelector`.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+type: labelSelector
+{{< /code >}}
+{{< code json >}}
+{
+  "type": "labelSelector"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+value        | 
+-------------|------ 
+description  | Selector expression the query will use to identify the events that the monitoring rule should process.
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+value: region == us-west-2 && cmpt == psql
+{{< /code >}}
+{{< code json >}}
+{
+  "value": "region == us-west-2 && cmpt == psql"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+#### Rules attributes
+
+template     | 
+-------------|------ 
+description  | Name of the [rule template][8] the component should use .
+required     | true
+type         | String
+example      | {{< language-toggle >}}
+{{< code yml >}}
+template: status-threshold
+{{< /code >}}
+{{< code json >}}
+{
+  "template": "status-threshold"
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+arguments    | 
+-------------|------ 
+description  | The [arguments][5] to pass to the [rule template][8] for the component.
+required     | false
+type         | Map of key-value pairs
+example      | {{< language-toggle >}}
+{{< code yml >}}
+arguments:
+  status: non-zero
+  threshold: 25
+{{< /code >}}
+{{< code json >}}
+{
+  "arguments": {
+    "status": "non-zero",
+    "threshold": 25
+  }
+}
+{{< /code >}}
+{{< /language-toggle >}}
+
+##### Arguments attributes
+
```

Comment:
I'm not sure we need this section -- wouldn't the arguments depend on whatever you've configured in the rule template? In other words, would it always be status and threshold? Are other argument attributes possible?